src/lib/test/integration/developer.test.ts

```
@@ -0,0 +1,1755 @@
+    // First, try to login directly
+    console.log(`Attempting login with email: ${TEST_DEVELOPER_EMAIL}`);
+    const response = await platformLogin(TEST_DEVELOPER_EMAIL!, TEST_DEVELOPER_PASSWORD!);
+    console.log("Login successful");
+    cachedLoginResponse = response;
+
+    // Store tokens for subsequent API calls
+    window.localStorage.setItem("access_token", response.access_token);
+    window.localStorage.setItem("refresh_token", response.refresh_token);
+
+    return response;
+  } catch (loginError) {
```

ESLint

[error] 'loginError' is defined but never used.

(@typescript-eslint/no-unused-vars)